actions/pypi_build/action.yaml

```diff
@@ -1,4 +1,4 @@
-name: "Build PyPi"
+name: "Build PyPi Wheel"
```

Comment:
let's change the name of this. calling it "pypi_build" is confusing, since it is not directly related to pypi. maybe just say "build_ml_repo" or something would be better.

Reply:
updated it to build-ml-whl.